test/passwords.js

```diff
@@ -0,0 +1,126 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+let userToken;
+before((done) => {
```

Comment:
No globals. Wrap all your definitions inside a describe block that describes this test file.

Reply:
whenever  i  put it inside a block my test starts failing, my test has started failing now

Reply:
token becomes undefined...

Reply:
done